hal/inc/usart_hal.h

```diff
@@ -30,13 +30,18 @@
 #include <stdint.h>
 #include <unistd.h>
 #include "platforms.h"
+#include "hal_platform.h"
 
 /* Includes ------------------------------------------------------------------*/
 // #include "pinmap_hal.h"
 
 /* Exported defines ----------------------------------------------------------*/
 
+#if HAL_PLATFORM_UART_BUFFER_ALIGNMENT > 4
```

Comment:
This shouldn't be needed anymore and in general due to `portBYTE_ALIGNMENT` being 32 on P2.